bpf/lib/lb.h

```diff
@@ -396,8 +402,15 @@ static __always_inline int reverse_map_l4_port(struct __ctx_buff *ctx, __u8 next
 				return ret;
 
 			if (port != old_port) {
+#ifdef ENABLE_SCTP
+				/* This will change the SCTP checksum, which we cannot fix right now.
+				 * This will likely need kernel changes before we can drop this.
+				 */
+				if (nexthdr == IPPROTO_SCTP)
+					return DROP_CSUM_L4;
+#endif  /* ENABLE_SCTP */
 				ret = l4_modify_port(ctx, l4_off, TCP_SPORT_OFF,
-						     csum_off, port, old_port);
+							csum_off, port, old_port);
```

Comment:
Seems this was changed by mistake?

Reply:
Yeah, indenting with tabs and spaces is always weird for me :(

Reply:
Indenting with tabs, aligning with spaces, *à la* Linux :slightly_smiling_face: